plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -547,6 +547,8 @@ func ClusterRoles() []rbacv1.ClusterRole {
 		rbacv1helpers.NewRule("create").Groups(authorizationGroup).Resources("subjectaccessreviews").RuleOrDie(),
 		// Needed for volume limits
 		rbacv1helpers.NewRule(Read...).Groups(storageGroup).Resources("csinodes").RuleOrDie(),
+		// Needed for namespaceSelector feature in pod affinity
+		rbacv1helpers.NewRule(Read...).Groups(legacyGroup).Resources("namespaces").RuleOrDie(),
```

Comment:
Shall we wrap it in a feature gate checking? similar to the CSIStorageCapacity checking below.

Reply:
I was thinking about that too, we could, but in any case this is part of the API PR.